### PR TITLE
Add meter reading occurrence rule

### PR DESCRIPTION
**Pull request by kft**

### CODE REVIEW

File: source/Energinet.DataHub.MeteringPoints.Domain/MeteringPoints/Consumption/CreationRules.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+using Energinet.DataHub.MeteringPoints.Domain.MeteringPoints.Consumption.Rules;
+
+namespace Energinet.DataHub.MeteringPoints.Domain.MeteringPoints.Consumption
+{
+    internal class CreationRules : MarketMeteringPoints.CreationRules
+    {
+        public CreationRules(MeteringPointDetails meteringPointDetails)
+            : base(meteringPointDetails)
+        {
+            Add(new PowerPlantIsRequiredForNetSettlementGroupRule(meteringPointDetails.GsrnNumber, meteringPointDetails.NetSettlementGroup, meteringPointDetails.PowerPlantGsrnNumber));
```
Comment (erduna) [inline source/Energinet.DataHub.MeteringPoints.Domain/MeteringPoints/Consumption/CreationRules.cs:25]: All except scheduled meter reading date are the same for production metering points. Should maybe be moved up to market metering point level.

File: source/Energinet.DataHub.MeteringPoints.Application/Validation/Rules/MeterReadingOccurenceRule.cs [hunk 2]
```diff
@@ -49,9 +48,12 @@ public MeterReadingOccurenceRule()
                     .WithState(createMeteringPoint => new MeterReadingOccurenceInvalidValueValidationError(createMeteringPoint.MeterReadingOccurrence));
             }).Otherwise(() =>
             {
-                RuleFor(request => request.MeterReadingOccurrence)
-                    .Must(meterReadingOccurrence => meterReadingOccurrence == ReadingOccurrence.Hourly.Name || meterReadingOccurrence == ReadingOccurrence.Quarterly.Name)
-                    .WithState(createMeteringPoint => new MeterReadingOccurenceInvalidValueValidationError(createMeteringPoint.MeterReadingOccurrence));
+                When(createMeteringPoint => createMeteringPoint.TypeOfMeteringPoint != MeteringPointType.Consumption.Name, () =>
```
[the remark this answers is not in the document]
Comment (kft) [reply source/Energinet.DataHub.MeteringPoints.Application/Validation/Rules/MeterReadingOccurenceRule.cs:51]: Only Consumption is considered here. So yes, it is correct for now.

File: source/Energinet.DataHub.MeteringPoints.Domain/MeteringPoints/Consumption/CreationRules.cs (hunk 1 above)
Comment (kft) [reply source/Energinet.DataHub.MeteringPoints.Domain/MeteringPoints/Consumption/CreationRules.cs:25]: Will be handles when Production mp is implemented.

**Review by thbaan — APPROVED** [review]

File: source/Energinet.DataHub.MeteringPoints.Application/Validation/Rules/MeterReadingOccurenceRule.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (erduna) [reply source/Energinet.DataHub.MeteringPoints.Application/Validation/Rules/MeterReadingOccurenceRule.cs:51]: even if we ignore everything but consumption, shouldn't we remove the ! ?

File: source/Energinet.DataHub.MeteringPoints.Domain/MeteringPoints/Consumption/CreationRules.cs (hunk 1 above)
Comment (erduna) [reply source/Energinet.DataHub.MeteringPoints.Domain/MeteringPoints/Consumption/CreationRules.cs:25]: OK, should we add a //TODO so we remember?

**Review by cmgrd — APPROVED** [review]

File: source/Energinet.DataHub.MeteringPoints.Domain/MeteringPoints/Consumption/CreationRules.cs (hunk 1 above)
Comment (kft) [reply source/Energinet.DataHub.MeteringPoints.Domain/MeteringPoints/Consumption/CreationRules.cs:25]: No. AC's will tell what to do :-)

**Pull request merged**